cypress/integration/learn/challenges/code-storage.js

```diff
@@ -0,0 +1,36 @@
+/* global cy */
+
+const selectors = {
+  defaultOutput: '.output-text',
+  editor: '.monaco-editor'
+};
+
+const location =
+  '/learn/responsive-web-design/basic-html-and-html5/say-hello-to-html-elements';
+
+describe('Challenge with editor', function () {
+  before(() => {
+    cy.visit(location);
+  });
+
+  it('renders seed code without localStorage', () => {
+    const editorContents = `<h1>Hello</h1>`;
+    cy.get(selectors.editor)
+      .contains(editorContents)
+      .type(`{movetoend}<h1>Hello World</h1>`);
```

Comment:
```suggestion
    cy.get(selectors.editor).as('editor').contains(editorContents);
    cy.get('@editor').click().focused().type(`{movetoend}<h1>Hello World</h1>`);
```
This way we're letting Monaco handle focus.